### ISSUE TITLE
Support multiple simple repeats

### ISSUE THREAD

**Issue by newlandsvalley** [newlandsvalley/purescript-abc-parser#28]
Simple (non-variant) repeats are only supported once.  The parser only allows a single colon, but the main impact will be on the ```Midi``` module (and ```abc-melody```) as is the case for improved variant endings.

**4.8 Repeat/bar symbols**

```
...
By extension, |:: and ::| mean the start and end of a section that is to be repeated three times, and so on.
...
```

**Comment by newlandsvalley** [comment]
Fixed by 0bfc47922db4cf7ca833b714a97b368babf12550 in the voltas branch

**Comment by newlandsvalley** [comment]
However, on investigating various music theory sites, I see absolutely no information about how multiple repeats (>=2) should be represented in a score.  The only example of a double colon I've seen comes from [abcnotation,com](http://abcnotation.com/tunePage?a=trillian.mit.edu/~jc/music/abc/src/jcabc2ps-20090401/abc/Repeat2D3/0000) which is worryingly self-referential.  Other sources say you use the single colon but mark it with a number above the stave to indicate the repetition count, or else abuse the volta notation.

Also, I presume that there is some sensible maximum for the number of such repeats which the parser should take note of.
